integration-tests/src/test/java/oracle/kubernetes/operator/ItManagedCoherence.java

```diff
@@ -221,20 +221,20 @@ public void testCreateCoherenceDomainInImageUsingWlst() throws Exception {
     LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - " + testMethodName);
   }
 
-  private void createDomainAndDeployApp(Map<String, Object> domainMap, String domainUID) throws Exception {
+  private void createDomainAndDeployApp(Map<String, Object> domainMap, String domainUid) throws Exception {
```

Comment:
I don't support spurious changes like this

Reply:
Yeah, I fixed the CheckStyle errors.  I'm not sure why these aren't being caught when others commit.  @ddsharpe, any ideas?

Reply:
I fully support changes like this.  This is technical debt resolution.  The original code did not meet our coding standard, and is therefore technical debt.  The code was originally allowed in because nobody caught it, and the checkstyle implementation was flawed.